ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -69,6 +72,9 @@ pub enum CoreError {
 
     #[error("Malformed ID input")]
     InvalidIDError(#[from] ParseIntError),
+
+    #[error("Invalid character in identifier: {0}")]
+    InvalidIdentifierError(String),
```

Comment:
this shouldn't be a core error.

You might refer [RegisterInput](https://github.com/TabbyML/tabby/blob/8c5b864ee7e64dc2a823d1bc505b2989920f5407/ee/tabby-webserver/src/service/auth.rs#L31) on how to add validation for service